recipes/monetdb/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "MonetDB" %}
+{% set version = "11.25.15" %}
+{% set sha256 = "8166268f676f75f0df7fdf3f8b2a49ffff422420d275bb994dddcab95012f89e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://www.monetdb.org/downloads/sources/Dec2016-SP3/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - toolchain
+    - pcre
+    - libxml2
+    - openssl
+    - libiconv
+  run:
+    - pcre
+    - libxml2
+    - openssl
+    - libiconv
+
+test:
+  commands:
+    - monetdb --help
+    - monetdbd --help
```

Comment:
Please add
```yaml
    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```
to the test section. Though you can probably ignore the osx one since this is a linux-only recipe.